tests/ra_aid/proc/test_interactive.py

```diff
@@ -51,25 +51,28 @@ def test_interactive_command():
     assert b"stderr" in output
     assert retcode == 0
 
-
 def test_large_output():
     """Test handling of commands that produce large output."""
     # Generate a large output with predictable content
     cmd = "for i in {1..10000}; do echo \"Line $i of test output\"; done"
     output, retcode = run_interactive_command(["/bin/bash", "-c", cmd])
-    
-    # Filter out script header/footer
-    lines = [line for line in output.splitlines() if b"Script" not in line and line.strip()]
-    
+
+    # Clean up specific artifacts (e.g., ^D)
+    output_cleaned = output.lstrip(b'^D')  # Remove the leading ^D if present
```

Comment:
Since everything seems to be working fine stripping the ^D, we can probably leave this as is.